mne_bids/mne_bids.py

```diff
@@ -302,16 +334,26 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
                             'acq_time': [acq_time]},
                       columns=['filename', 'acq_time'])
 
-    df.to_csv(fname, sep='\t', index=False, na_rep='n/a')
-
-    if verbose:
-        print(os.linesep + "Writing '%s'..." % fname + os.linesep)
-        print(df.head())
+    if os.path.exists(fname) and write_mode != 'overwrite':
+        orig_df = pd.read_csv(fname, sep='\t')
+        # if the file name is already in the file raise an error if needed
+        if write_mode == 'error':
+            if raw_fname in orig_df['filename'].values:
+                raise OSError(errno.EEXIST, '"%s" already exists in the'
+                              'scans list. Please set write_mode to '
+                              '"overwrite" or "append".' % raw_fname)
+        # otherwise add the new data, dropping any duplicates.
+        df = orig_df.append(df)
+        df.drop_duplicates(subset='filename', keep='last', inplace=True)
+        df = df.sort_values(by='acq_time')
```

Comment:
is this really necessary for BIDS compatibility? Otherwise, I'd skip the sorting ...

Reply:
also how would the sorting work if some values are `n/a`?

Reply:
no, not necessary, I just thought it was nice. I added it before there was any mention of wanting to get rid of pandas so I figured why not try and make the data as easy to read as possible.
And pretty sure the `n/a` values get sorted to the bottom iirc.

Reply:
I think that sorting is a good idea because these files will be read by humans :-)

regarding the removal of pandas --> seeing this PR, I am getting more convinced that we should keep it.

Reply:
okay let's keep it for now and we can move this discussion to the `pandas` issue.